_includes/cockroachcloud/sql-connection-string-free.md

```diff
@@ -1,20 +1,28 @@
     <section class="filter-content" markdown="1" data-scope="mac">
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    cockroach sql --url 'postgresql://<user>@<free-tier-host>.<region>.cockroachlabs.cloud:26257/defaultdb?sslmode=verify-full&sslrootcert='$HOME'/.postgresql/root.crt&options=--cluster=<cluster-name>-<tenant-id>'
+    cockroach sql --url 'postgresql://<username>:<password>@<serverless-host>:26257/defaultdb?sslmode=verify-full&sslrootcert='$HOME'/.postgresql/root.crt&options=--cluster=<cluster-id>'
```

Comment:
Just fyi the connection string change to `serverless-host` isn't done yet, but I think it will make it into the launch. https://cockroachlabs.atlassian.net/browse/CC-4970